### DISCUSSION TITLE
Observer Enhancements

### DISCUSSION THREAD

**Discussion by deathreaperz** [Phobos-developers/Phobos#1044] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

Allow Observer to see cloaked units and disguised unit.

### INI code

```ini
[General]
Observer.SeeCloaked=no ;bool, whether allow observer see all cloaked/submerged units
Observer.SeeDisguised=no ;bool, whether allow observer see all disguised infantry/units
```

**Comment by Starkku** [comment]
Displaying cloaked objects was implemented in commit db67a1fb785a36149116294878a7a63b5bea66d7 in `develop` branch.
Displaying disguised objects is implemented as part of #653.

Both are enabled by default with no way toggle it off.
